web/src/app/icons/components/Icons.js

```diff
@@ -26,7 +26,7 @@ export class Trash extends Component {
 @withStyles(styles)
 export class Warning extends Component {
   render() {
-    const { classes, details } = this.props
+    const { classes, tooltip } = this.props
```

Comment:
Would calling it `text`, `message` or `label` be more appropriate? Also let's add a string prop-type for it.